bin/vagrant_provision.sh

```diff
@@ -69,7 +69,6 @@ VENV=/home/vagrant/virtualenv
 sudo -H -u vagrant -s -- <<EOF
 virtualenv $VENV
 source $VENV/bin/activate
-pip install -U "pip<7"
```

Comment:
The vagrant box comes with a fixed version of Pip that doesn't change. If we change the base box for virtualbox, it may come with a newer version of Pip. If someone is changing the base box, they should hit a problem here.